roles/ciao-compute/README.md

```diff
@@ -1,33 +1,36 @@
-ciao-compute
-=========
+# ciao-compute
 Ansible role to install the compute node for a CIAO cluster
 
-Requirements
-------------
-None
+## Requirements
+- A previously setup of a CIAO controller node.
+  You can setup one with `ciao-controller` role
```

Comment:
I guess it would be nice to point the role you mean here, something like:

`You can setup one with [ciao-controller](https://github.com/clearlinux/clear-config-management/tree/master/roles/ciao-controller) role`

that would give:

You can setup one with [ciao-controller](https://github.com/clearlinux/clear-config-management/tree/master/roles/ciao-controller) role

which may be easier to browse if you're looking at the documentation